test/transformer.spec.js

```diff
@@ -206,6 +206,23 @@ describe( 'transformer', () => {
         } );
     } );
 
+    describe( 'arbitrary HTML', () => {
```

Comment:
Is this backfilling existing functionality or testing a change made here?

Reply:
This actually came up in discussion with @lognaturel where we were trying to determine the scope of where URLs might be present and need to be properly escaped, and in turn to determine the scope of testing for those cases. We felt it would be a good general addition so we

1. Know what the behavior actually is
2. Ensure we don't introduce regressions which could potentially cause HTML injection/XSS where it was previously not allowed